core/src/main/java/org/elasticsearch/search/aggregations/metrics/tophits/TopHitsAggregatorFactory.java

```diff
@@ -79,6 +79,7 @@ public TopHitsAggregatorFactory(String name, Type type, int from, int size, bool
     public Aggregator createInternal(Aggregator parent, boolean collectsFromSingleBucket, List<PipelineAggregator> pipelineAggregators,
             Map<String, Object> metaData) throws IOException {
         SubSearchContext subSearchContext = new SubSearchContext(context.searchContext());
+        subSearchContext.parsedQuery(context.searchContext().parsedQuery());
```

Comment:
is this related to the inner hits changes? I have no problem with it,I'm just curious if the top_hits somehow ties in with inner_hits